apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -152,6 +153,14 @@ class CensusForm extends Component {
 
   processError(error) {
     console.log(JSON.stringify(error, null, 2));
+    if (error.responseJSON.email_s[0] === "invalid") {
```

Comment:
Please add a brief comment before this method explaining that we're validating this one field on the server, including the reason (presumably because the logic is somewhat complex and we don't want to duplicate it on the client), but that all other validation is done on the client, because for simple validations its easier to do this all within React.